### PR TITLE
iOS, tvOS, watchOS

### PR DESCRIPTION
**Pull request by jpalao**

### CODE REVIEW

File: ios/config/Policy.sh [hunk 1]
```diff
@@ -0,0 +1,165 @@
+#!/bin/sh
```
Comment (Leont) [inline ios/config/Policy.sh:1]: I don't think this file should have been committed?

**Comment by Leont** [comment]
This branch could use a little more description.

But (assuming that it works), it sounds good :-)

**Comment by jkeenan** [comment]
> This branch could use a little more description.
> 
> But (assuming that it works), it sounds good :-)

Before we accept  this into the core distribution, we have to ask:  Can we maintain it?  Can we get smoke-test reports?

Thank you very much.
Jim Keenan

**Comment by jkeenan** [comment]
@jpalao you should be aware that perl-5.30 is now maintenance only.  It is highly unlikely that new functionality, such as what you propose here, will be added.

perl-5.32.0 was released today and is now in the process of disseminating to CPAN mirrors around the world.  You should consider rebasing your branch on that release.

Thank you very much.
Jim Keenan

**Comment by jpalao** [comment]
created new PR 
https://github.com/Perl/perl5/pull/17880

**Pull request closed without merging**